packages/ember-runtime/lib/system/string.js

```diff
@@ -54,7 +54,11 @@ var STRING_CAMELIZE_REGEXP = (/(\-|_|\.|\s)+(.)?/g);
 var STRING_UNDERSCORE_REGEXP_1 = (/([a-z\d])([A-Z]+)/g);
 var STRING_UNDERSCORE_REGEXP_2 = (/\-|\s+/g);
 
+var strFmtDeprecationWarning = 'Use of Ember.String.fmt() is deprecated. Use ES6 template strings instead: https://babeljs.io/docs/learn-es6/#template-strings';
+
 function fmt(str, formats) {
+  //Ember.deprecate('Use of Ember.String.fmt() is deprecated. Use ES6 template strings instead: https://babeljs.io/docs/learn-es6/#template-strings');
+  Ember.warn(strFmtDeprecationWarning);
```

Comment:
Yeah, I figured that should be the case. I changed it to Ember.deprecate and tried to chase it down (mostly just searching for `fmt` in `packages`) but I just don't know enough about Ember or the code base to track it down. Could you or someone walk me through some of the failing tests?